src/java/org/apache/cassandra/db/compaction/CompactionTask.java

```diff
@@ -301,6 +347,7 @@ public abstract class CompactionOperation implements AutoCloseable, CompactionPr
         final Set<SSTableReader> actuallyCompact;
         private final int fullyExpiredSSTablesCount;
         private final long inputDiskSize;
+        private final long inputUncompressedSize;
```

Comment:
can we please also add the "start time" to CompactionProgress ? my need is to expose it in the TableOperations